<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchProcessRawSessions</class>
 <widget class="QMainWindow" name="BatchProcessRawSessions">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>255</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Batch Process Raw Sessions</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../brl_logo.gif</normaloff>../brl_logo.gif</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>531</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Data settings</string>
    </property>
    <widget class="QPushButton" name="btnProcess">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>180</y>
       <width>70</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Process</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSaveSel">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>70</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtSaveDir">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Select the location of the AdaptiveLSV.npz file</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Where to output the resultant masks</string>
     </property>
     <property name="text">
      <string>Save Directory</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Where to output the resultant masks</string>
     </property>
     <property name="text">
      <string>Weights (*.h5)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnWeightsSel">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>110</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtWeightsDir">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Select the location of the AdaptiveLSV.npz file</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>The top level directory containing all the sessions</string>
     </property>
     <property name="text">
      <string>Root Directory</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRootSel">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>30</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtRootDir">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Select the location of the AdaptiveLSV.npz file</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="grpBoxLSOverride">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This setting is only respected in Adaptive PS mode&lt;br/&gt;For example used when all 8 lights have been used to capture, but you only need  4 corresponding to NIR, you would select NIR.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="title">
      <string>Image Datatype</string>
     </property>
     <widget class="QRadioButton" name="rdoGrayscale">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Grayscale</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rdoAlbedo">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Albedo</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rdoComposite">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Composite</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>edtSaveDir</tabstop>
  <tabstop>btnSaveSel</tabstop>
  <tabstop>btnProcess</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
